test/commands-tests/deploy.js

```diff
@@ -84,6 +84,19 @@ describe('ForgAE Deploy', () => {
             assert.equal(deployer.network.url, expectedNetwork)
         })
 
+        it('Should init Deployer with custom network', async () => {
+            // Arrange
+            const expectedNetwork = "192.168.99.100:3001"
+            const passedNetwork = "192.168.99.100:3001"
```

Comment:
maybe 'expectedNetwork' and 'passedNetwork' should be one variable, just 'network' ?